src/utils/apiRequests/api.ts

```diff
@@ -130,15 +134,15 @@ export async function postRequest(url: any, data: any) {
   return result;
 }
 
-export async function deleteAuthenticatedRequest(url: any, token: any) {
+export async function deleteAuthenticatedRequest(url: any) {
```

Comment:
create a new function called as deleteRequest and restore this function as it might be needed somewhere else